release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -37,8 +37,62 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### `lang.xml:data()` Lang Library Function
+
+The `lang.xml:data()`  lang library function has been introduced aiming to replace the usage of the `lang.xml:getContent()` function to get the character data from an `xml` value. This function returns the character data of a `xml:Text` value as a string. For an `xml:Element` value this returns the concatenation of the result of invoking this function over the children of the element. Calling `lang.xml:data()` on comments, processing instructions, and empty sequences returns an empty string.
```

Comment:
```suggestion
The `lang.xml:data()`  lang library function has been introduced aiming at replacinging the usage of the `lang.xml:getContent()` function to get the character data from an `xml` value. This function returns the character data of an `xml:Text` value as a string. For an `xml:Element` value, this returns the concatenation of the result of invoking this function over the children of the element. Calling `lang.xml:data()` on comments, processing instructions, and empty sequences returns an empty string.
```

Reply:
@rdhananjaya do we need the part about `lang.xml:getContent()` here? It is only for text that it is no longer available, right? Can't we just say

How about
```suggestion
The `lang.xml:data()`  lang library function has been introduced to get the character data from an `xml` value. This function returns the character data of an `xml:Text` value as a string. For an `xml:Element` value this returns the concatenation of the result of invoking this function over the children of the element. Calling `lang.xml:data()` on comments, processing instructions, and empty sequences returns an empty string.
```